### PR TITLE
Modified Moderated.sol to allow multiple moderators

### PR DESCRIPTION
**Pull request by matthew-ho**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a feature/bugfix branch** (right side). Don't request from master!
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Description
Allow multiple moderators (governors) which will be necessary when we want to have the register page for governors

### Checklist
* [x] Have you done your changes on a separate branch. Branches MUST have descriptive names that start with either the `fix/` or `feature/` prefixes. Good examples are: `fix/signin-loop` or `feature/pr-templates`.
* [x] Have you got descriptive commit messages that would make sense if prefixed with "This commit will ..."
* [x] Will you squash when you merge this PR?

#### New Features

1. [ ] Does your submission have a high coverage of your feature?
2. [x] Have you lint your code locally prior to submission?

💔Thank you!

### CODE REVIEW

File: contracts/Moderated.sol [hunk 1]
```diff
@@ -9,20 +9,20 @@ import "@openzeppelin/contracts/ownership/Ownable.sol";
...
 
   modifier onlyModerator() {
-    require(msg.sender == moderator, "Only a moderator can do this");
+    require(moderators[msg.sender], "Only a moderator can do this");
     _;
   }
 
   constructor() public Ownable() {
-    moderator = owner();
+    moderators[owner()] = true;
```
Comment (matthew-ho) [inline contracts/Moderated.sol:20]: I don't know if this should be empty or not as assuming we are the owners of the contract originally, should we be moderators

Comment (JamesDalboth) [reply contracts/Moderated.sol:20]: I think its a reasonable the owner starts off as a moderator

Comment (JamesDalboth) [reply contracts/Moderated.sol:20]: especially since only a moderator can add another moderator

**Review by JamesDalboth — COMMENTED** [review]
I've noticed you've gone for a mapping of address to bool rather than just having a list of address that are approved. Is this for a solidity reason?

File: contracts/Moderated.sol (hunk 1 above)
Comment (Tooni) [reply contracts/Moderated.sol:20]: i think it's ok. also it is easy to change in future.

**Review by Tooni — APPROVED** [review]

**Pull request merged**